app/templates/style_studio.html

```
@@ -94,35 +94,58 @@ <h2 class="fw-bold mb-1">Pick a hairstyle</h2>
+
+                        <!-- AI Reasoning Panel -->
+                        <div id="ai-reasoning-panel" class="rounded-4 border border-secondary border-opacity-25 mb-3 px-3 py-3 d-flex align-items-center"
+                            style="background-color: rgba(28, 28, 30, 0.95); backdrop-filter: blur(8px); min-height: 88px; transition: all 0.2s ease;">
+
+                            <div id="ai-panel-empty" class="d-flex align-items-center gap-2 text-secondary w-100 justify-content-center">
+                                <span style="color: var(--th-yellow); font-size: 1.1rem;">✨</span>
+                                <span class="text-sm">Hover a recommended style to see why it was suggested for you.</span>
+                            </div>
+
+                            <div id="ai-panel-content" class="d-none align-items-center gap-3 w-100">
+                                <img id="ai-panel-thumb" src="" alt="" class="rounded-3 flex-shrink-0" style="width: 64px; height: 64px; object-fit: cover;">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)